server/services/geocodes.js

```diff
@@ -0,0 +1,52 @@
+const { dbClient, collections } = require('../db');
+
+const getPointsFromPostalCodes = async (postalCodes) => {
+  const queryPoint = async (postalCode) => {
+    const spacedCode = postalCode.includes(' ')
+      ? postalCode
+      : `${postalCode.substring(0, 3)} ${postalCode.substring(3)}`;
+    const request = await dbClient.db[collections.GEOCODES].findOne({
+      postal_code: spacedCode,
+    });
+
+    if (request) {
+      // Happy path, full match on postal code
+      return {
+        lat: request.latitude,
+        lng: request.longitude,
+        match: request.postal_code.replace(' ', ''),
+      };
+    }
+    const approxRequest = await dbClient.db[collections.GEOCODES].findOne({
```

Comment:
this is a style nitpick but I would move the newline above this, as well as the next comment explaining what this branch of the code is doing :) just keeps things a little more nicely broken up

Reply:
like the comment from line 25 becomes line 21, line 20 is blank, and this line (20) moves to line 22